player/command.c

```diff
@@ -2280,6 +2280,7 @@ static int property_imgparams(struct mp_image_params p, int action, void *arg)
             (desc.flags & MP_IMGFLAG_ALPHA) ? MP_ALPHA_STRAIGHT : MP_ALPHA_AUTO;
     }
 
+    bool has_crop = mp_rect_w(p.crop) > 0 && mp_rect_h(p.crop) > 0;
```

Comment:
I don't think this works? It seems to always give me values even when I'm not cropping anything.

Reply:
Full frame crop is still valid and is treated as such. Does is pose any problem?

Reply:
I'm not using `--video-crop` at all though. I thought the intention was to make the property unavailable in that case.

Reply:
It is not thigltly connected to `--video-crop` alone. It can come from decoder/container and if not it is set to full frame
```
    dst->params.crop.x0 = src->crop_left;
    dst->params.crop.y0 = src->crop_top;
    dst->params.crop.x1 = src->width - src->crop_right;
    dst->params.crop.y1 = src->height - src->crop_bottom;
```

It just flows with the code. Only reason to exclude this property is when it is set to 0 0 0 0 to avoid checking on user side for the values. But else (0 0 w h) is valid crop.

Reply:
Oh okay, I guess I just misunderstood.